lib/node_modules/@stdlib/math/base/special/besselj1/test/test.js

```diff
@@ -0,0 +1,270 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var j1 = require( './../lib' );
+
+
+// FIXTURES //
+
+var hugePositive = require( './fixtures/julia/huge_positive.json' );
+var veryLargePositive = require( './fixtures/julia/very_large_positive.json' );
+var largePositive = require( './fixtures/julia/large_positive.json' );
+var mediumPositive = require( './fixtures/julia/medium_positive.json' );
+var smallPositive = require( './fixtures/julia/small_positive.json' );
+var smallNegative = require( './fixtures/julia/small_negative.json' );
+var smaller = require( './fixtures/julia/smaller.json' );
+var tinyPositive = require( './fixtures/julia/tiny_positive.json' );
+var subnormal = require( './fixtures/julia/subnormal.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof j1, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function evaluates the Bessel function of the first kind of order one (J_1) of `x` on the interval `[50,500]`', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	expected = veryLargePositive.expected;
+	x = veryLargePositive.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = j1( x[i] );
+		if ( y === expected[i] ) {
+			t.equal( y, expected[i], 'x: '+x[i]+', y: '+y+', expected: '+expected[i] );
+		} else {
+			delta = abs( y - expected[i] );
+			tol = EPS * abs( expected[i] ) * 600.0;
```

Comment:
Can you put the scale factor first? e.g., `600.0 * EPS`. Doing so makes it easier to scan by eye the tolerance magnitude and also read `600eps` LTR, rather than RTL with noise in between. I have already addressed this in `j0`.